spec/lib/client/middleware_spec.rb

```diff
@@ -71,10 +71,10 @@ def perform(_)
 
     it 'enqueues previously scheduled job' do
       QueueWorker.sidekiq_options unique: true
-      QueueWorker.perform_in(60 * 60, [1, 2])
+      jid = QueueWorker.perform_in(60 * 60, 1, 2)
```

Comment:
This test wasn't testing the same job moving from the scheduled to its normal queue. The signature for the perform_in method is ` def perform_in(interval, *args)` https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/worker.rb#L45  Therefore the first job had a single argument of an array and the second job had two Fixnum arguments. 

Removing the brackets caused the test to fail identifying issue #101